airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -446,7 +446,7 @@ async def _run(self, current_acceptance_tests_result: StepResult) -> StepResult:
 
 
 class LiveTestSuite(Enum):
-    ALL = "all"
+    ALL = "live"
```

Comment:
I"m not sure I get the reason of this change

Reply:
I just didn't like the way the name was displayed in the report ("Running Connector All Tests") - it wasn't clear what exactly was running.